migrate/scripts/watertower-tooltip.js

```diff
@@ -0,0 +1,17 @@
+const db  = require(../db.js)
```

Comment:
Does it work to not pass a string to `require`?

Reply:
Nope, typo there :)